File: Java code/src/main/java/VIA/VIATalks/Database/data/Ticket.java
package VIA.VIATalks.Database.data;

import java.io.Serializable;

public class Ticket implements Serializable {
    private int id;
    private String ticketNumber;

    //Constructor
    public Ticket(int id, String ticketNumber) {
        this.id = id;
        this.ticketNumber = ticketNumber;
    }

    //Getters and Setters

    public int getId() {
        return id;
    }

    public String getTicketNumber() {
        return ticketNumber;
    }

    public void setTicketNumber(String ticketNumber) {
        this.ticketNumber = ticketNumber;
    }
}

File: C# code/VIATalks/Shared/Ticket.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Shared
{
    [Serializable]
    class Ticket
    {
        public int Id { get; set; }
        public string TicketNumber { get; set; }

        public Ticket(int id, string ticketNumber)
        {
            Id = id;
            TicketNumber = ticketNumber;
        }
    }
}
